en/flight_controller/pixhawk4.md

```diff
@@ -1,8 +1,8 @@
 # Pixhawk 4 
 
-Pixhawk<sup>&reg;</sup> 4 is an advanced autopilot designed and made in collaboration with Holybro<sup>&reg;</sup> and the PX4 team. It is optimized to run PX4 version 1.7, suitable for academic and commercial developers. 
+*Pixhawk 4*<sup>&reg;</sup> is an advanced autopilot designed and made in collaboration with Holybro<sup>&reg;</sup> and the PX4 team. It is optimized to run PX4 version 1.7, suitable for academic and commercial developers. 
```

Comment:
"It is optimized to run PX4 version 1.7, suitable for academic and commercial developers" will date badly - ie when 1.8 and later are released. SOmething to think about, not necessarily change.